fastlane_core/lib/fastlane_core/project.rb

```diff
@@ -172,6 +172,14 @@ def app_name
       return "App" # default value
     end
 
+    def command_line_library?
+      return true if build_settings(key: "PRODUCT_TYPE") == "com.apple.product-type.library.dynamic"
```

Comment:
`"com.apple.product-type.library.dynamic"` is this the product type when Xcode is building a dynamic framework also? I'm wondering if we should only worry about supporting `"com.apple.product-type.tool"` below?